### PR TITLE
Fix verbose travis

### PR DESCRIPTION
**Pull request by colincadams**

### CODE REVIEW

File: travis/deploy_staging.sh [hunk 1]
```diff
@@ -1,32 +1,37 @@
-#!/bin/bash set -v
-#
 # Use verbose mode so that the commands are printed to Travis. Do not expand
-# variables as some are secrets.
+# variables (set -x) as some are secrets.
```
Comment (jessex) [inline travis/deploy_staging.sh]: -x or -v?

Comment (colincadams) [reply travis/deploy_staging.sh]: set -x would expand them, so we want to use set -v instead. would different wording help?

Comment (jessex) [reply travis/deploy_staging.sh]: My first reading of the comment was that you were saying `set -x` is a thing we are doing or should do, but if you're saying that `set -x` is a thing we should absolutely never do, I think we should be _slightly_ more explicit. Or I could just be overly pedantic. That wouldn't be new.

Comment (colincadams) [reply travis/deploy_staging.sh]: done!

**Pull request merged**